LEGO1/lego/legoomni/include/legorace.h

```diff
@@ -82,6 +82,7 @@ class RaceState : public LegoState {
 	Entry* GetState(MxU8 p_id);
 
 	undefined4 GetUnknown0x28() { return m_unk0x28; }
+	void SetUnknown0x28(undefined4 p_unk0x28) { m_unk0x28 = p_unk0x28; }
```

Comment:
Are we sure that those setters/getters are used? In my experience they are often not - any hint from `BETA` in this regard? If the code still matches we should access the members directly

Reply:
Good catch! I have an indication from BETA10 0x100c81cd:
```c++
*(undefined4 *)((this->base).m_unk0x10c + 0x28) = 2;
```

I forgot that many getters in the code are not verified - the presence of the getter made be believe that a setter should be used.

Reply:
Yeah, I think the `*State` classes generally don't use getters/setters (with few exceptions), we gotta remove the ones we already set up (that was before we got hands on the beta)